src/components/AllToDos.tsx

```diff
@@ -0,0 +1,23 @@
+import SingleToDo from "./SingleToDo";
+
+type propsType= {
```

Comment:
the naming convention of the types follows PascalCase so it should be **PropsType** and we usually name it as the component name + Props
e.g: AllTodosProps